src/index.html

```diff
@@ -3,11 +3,402 @@
+        </h2>
+
+        <section class="by-category__wrapper">
+          <img
+            src="./images/photos/1.png"
+            alt="Earbuds usage"
+            class="by-category__earbuds"
+          >
+
+          <img
+            src="./images/photos/2.png"
+            alt="Headphones usage"
+            class="by-category__headphones"
+          >
+
+          <p class="by-category__name">
+            Headphones & earbuds
+          </p>
+        </section>
+
+        <section class="by-category__wrapper">
+          <img
+            src="./images/photos/3.png"
+            alt="Light blue speaker"
+            class="by-category__speaker-light-blue"
+          >
+
+          <img
+            src="./images/photos/4.png"
+            alt="Headphones usage"
+            class="by-category__speaker-blue"
+          >
+
+          <p class="by-category__name">
+            Speakers
+          </p>
+        </section>
+
+        <section class="by-category__wrapper">
+          <img
+            src="./images/photos/5.png"
+            alt="Blue audio sunglasses"
+            class="by-category__sunglasses-blue"
+          >
+
+          <img
+            src="./images/photos/6.png"
+            alt="Black audio sunglasses"
+            class="by-category__sunglasses-black"
+          >
+
+          <p class="by-category__name">
+            Audio sunglasses
+          </p>
+        </section>
+
+        <div class="by-category__button-wrapper">
+          <a href="#" class="by-category__button button">
+            View all
+          </a>
+        </div>
+      </article>
+
+      <article class="testimonials">
+        <section class="testimonials__wrapper">
+          <h2
+            class="title testimonials__title"
+            id="testimonials"
+          >
+            Why buy direct from Bose
+          </h2>
+
+          <div class="testimonials__text">
+            <p class="testimonials__line">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="testimonials__line">
+              90-day risk-free trial
+            </p>
+
+            <p class="testimonials__line">
+              World class customer service
+            </p>
+
+            <p class="testimonials__line">
+              My Bose account management
+            </p>
+
+            <p class="testimonials__parag">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+            </p>
+          </div>
+        </section>
+      </article>
+
+      <article class="contact">
+        <h2
+          class="contact__title title"
+          id="contact"
+        >
+          Contact us
+        </h2>
+
+        <section class="contact__wrapper">
+          <div class="contact__form-wrapper">
+            <form
```

Comment:
checklist suggests
>15 Page shouldn't be reloaded on form submit:

currently it does reload the page. You could try js or a more simple inline html approach:
`<form onsubmit="document.querySelector('<form_selector>').reset()"`

Reply:
so that form validates input, and responds to submit action, yet not reloading the page